source/overview/product.rst

```diff
@@ -57,7 +56,9 @@ To get started, `download the open source Mattermost Team Edition server <https:
 
 Get started with a `free trial license <https://about.mattermost.com/trial/>`__. You can later purchase online with a `credit card <https://about.mattermost.com/pricing/>`__ or by `bank transfer <https://about.mattermost.com/quotation/>`__.
 
-**Mattermost Enterprise Edition E20** is commercial enterprise-grade messaging system that scales to hundreds to tens of thousands of users. E20 is your "virtual city", with cluster-based horizontal scaling, high availability deployment, performance monitoring systems and compliance reporting plus advanced corporate directory integration. It offers the following advanced features **in addition to** E10:
+Mattermost Enterprise Edition E20
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A commercial enterprise-grade messaging system that scales from hundreds to tens of thousands of users. E20 is your "virtual city", with cluster-based horizontal scaling, high availability deployment, performance monitoring systems and compliance reporting plus advanced corporate directory integration. Building on the features in E10, it offers the following advanced features:
 
 - Advanced SAML 2.0 authentication with Okta, OneLogin, and Active Directory Federation Services 
```

Comment:
I would add the following to the list 

```
 - Custom retention policies for messages and files
 - Active Directory / LDAP Group sync
 - Elasticsearch support for highly efficient database searches in a cluster environment
```

Retention policies above or below compliance exports; LDAP group sync below SAML; elasticsearch below horizontal scaling.